docs/src/schedule_synthesis.md

```diff
@@ -0,0 +1,6 @@
+# Schedule Synthesis
+
```

Comment:
We need actual documentation here, beyond simply giving the function docstrings!  Please give a short explanation of what this portion of the package is for, and how to use these functions.